app/code/Magento/Ui/view/base/web/js/modal/modal.js

```diff
@@ -305,6 +305,9 @@ define([
          * * @return {Element} - current element.
          */
         closeModal: function () {
+            if (!this.options.isOpen) {
+                return this.element;
+            }
             var that = this;
```

Comment:
According to Magento code style guideline, the variables declaration should be in the top of the function. Please move `var` declaration to the top of the function.